spec.md

```diff
@@ -603,6 +603,19 @@ interface Literal <: Node, Expression {
 
 A literal token. Note that a literal can be an expression.
 
+### RegexLiteral
+
+```js
+interface RegexLiteral <: Literal {
+  regex: {
+    pattern: string;
+    flags: string;
+  };
+}
+```
+
+The `regex` property allows regexes to be represented in environments that don’t
```

Comment:
Should there be a note saying that the `value` property can be `null` in such cases?